azure-toolkit-libs/azure-toolkit-sqlserver-lib/src/main/java/com/microsoft/azure/toolkit/lib/sqlserver/model/SqlFirewallRuleEntity.java

```diff
@@ -6,6 +6,7 @@
 package com.microsoft.azure.toolkit.lib.sqlserver.model;
 
 import com.microsoft.azure.toolkit.lib.common.entity.IAzureResourceEntity;
+import com.microsoft.azure.toolkit.lib.common.utils.MacUtils;
```

Comment:
how did you assign responsibilities between `MacUtils`/`HashMacUtils`/`NetUtils`?